### PR TITLE
Add slug field to menu

### PR DESCRIPTION
**Pull request by tomaszszymanski129**
I want to merge this change because it adds slug field to menu model and related mutations.

Linked task: [SALEOR-1346](https://app.clickup.com/t/2549495/SALEOR-1346)

# Impact

* [X] New migrations
* [X] New/Updated API fields or mutations
* [X] Deprecated API fields or mutations
* [ ] Removed API types, fields, or mutations
* [ ] Documentation needs to be updated

# Pull Request Checklist

* [ ] Privileged queries and mutations are guarded by proper permission checks
* [ ] Database queries are optimized and the number of queries is constant
* [ ] Database migration files are up to date
* [ ] The changes are tested
* [X] GraphQL schema and type definitions are up to date
* [ ] Changes are mentioned in the changelog

### CODE REVIEW

**Comment by patrys** [comment]
Task linked: [SALEOR-1346 Add slug to menus](https://app.clickup.com/t/2549495/SALEOR-1346)

File: saleor/graphql/core/validators.py [hunk 1]
```diff
@@ -9,16 +9,22 @@
     from decimal import Decimal
 
 
-def validate_one_of_args_is_in_query(arg1_name, arg1, arg2_name, arg2):
-    if arg1 and arg2:
+def validate_one_of_args_is_in_query(*args):
+    # split args into 2-element tuples
```
Comment (IKarbowiak) [inline saleor/graphql/core/validators.py:13]: For me, it was hard to understand what do you want to achieve. IMO more detailed description will help a lot. Like:
```suggestion
    # split args into a list with 2-element tuples: [(arg1_name, arg1_value), (arg2_name, arg2_value), ...]
```

**Review by IKarbowiak — APPROVED** [review]

**Review by fowczarek — APPROVED** [review]
Should we add `slug` to `menu_fields` in `MenuFilter`?

**Review by maarcingebala — APPROVED** [review]

**Comment by maarcingebala** [comment]
@fowczarek That's a good idea but I would add a filter that accepts a list of slugs:
```
input MenuFilterInput {
  slug: [String!]
}
```
Then you could easily get all your menus in one query.

**Comment by fowczarek** [comment]
@maarcingebala I think that we should add both changes. In other `search` fields we searching by `name` and `slug`.

**Comment by tomaszszymanski129** [comment]
@fowczarek @maarcingebala please check last commit where I've added proposed filtering possibilities.

**Review by maarcingebala — APPROVED** [review]

**Review by fowczarek — APPROVED** [review]

**Pull request merged**